DOC.md

```diff
@@ -1501,6 +1502,104 @@ callback will have access to the `POSTFIX_MATCH` field as well.
         }
 ```
 
+## Snippet List
+
+Makes an `open` function available to use to open currently available snippets
+in a different buffer/window/tab.
+
+An `options` table with a `display` option (function) is also available in
+order to customize how you want the buffer/window/tab to look like. We'll take
+a deeper dive in the example below.
```

Comment:
I'd prefer a more structured approach here, to improve readability.
So
* show the function-prototype
* list the arguments and possible options to those arguments

[example here](https://github.com/L3MON4D3/LuaSnip/blob/7bfae6fbedafa5572b5289b3dc2cba4cfde2039c/DOC.md?plain=1#L1219-L1240)

Reply:
Oh, and explain `open` and the default-display separately, I think that will also make it more concise

Reply:
I addressed this in https://github.com/L3MON4D3/LuaSnip/pull/652/commits/34c03fa8c8e6836b13232111c7e33f98a015f881!